components/ui/AppCta.vue

```diff
@@ -37,5 +37,26 @@ export default class extends Vue {
   & :first-child {
     margin-right: $layout-02;
   }
+  svg {
+      overflow: visible;
+  }
+  &[target=_blank] {
```

Comment:
Now, let's rely on the `isExternal` computed property to dynamically add BEM Block Modifiers depending on the property. Find the root of the template and use the object syntax of the `class` property to add modifiers depending of it is external or internal. For instance:

```
:class="{
  'app-cta_is-external': isExternal,
  'app-cta_is-internal': !isExternal
}"
```

Reply:
Once you have those modifiers, structure the SASS code in two blocks, one for when the block is internal, and other for when it is external:

```
.app-cta {
  ...
  &_is-internal {
  }

  &_is-external {
  }
}
```